tools/cli/commands/changelog.js

```diff
@@ -523,7 +523,7 @@ async function changelogArgs( argv ) {
 	argv = await validateProject( argv );
 	argv.success = `Command '${ argv.cmd || argv._[ 1 ] }' for ${
 		argv.project
-	} completed succesfully!`;
+	} completed successfully!`;
```

Comment:
Good change, but doesn't seem to have anything to do with the PHP 7.2 or WP changes.

Reply:
Updated PR title.